projects/dxilconv/test/taef/lit.site.cfg.in

```diff
@@ -7,6 +7,7 @@ config.llvm_obj_root = "@LLVM_BINARY_DIR@"
 config.llvm_build_mode = "@LLVM_BUILD_MODE@"
 config.te = "@TAEF_EXECUTABLE@"
 config.taef_arch = "@TAEF_ARCH@"
+config.verbose = True
```

Comment:
Do we need this in the site config or can this be based on the lit tool verbosity option?

Reply:
This verbose mode is for the TAEF tests and it is different from the lit tool verbose mode. We always want to have it on to make sure we capture full TAEF test output. Lit will show the output when the tests fails irrespective of the lit verbose mode.